tutorials/plugins/gdnative/files/cpp_example/SConstruct

```diff
@@ -50,10 +53,10 @@ if env['platform'] == "osx":
     env['target_path'] += 'osx/'
     cpp_library += '.osx'
     if env['target'] in ('debug', 'd'):
-        env.Append(CCFLAGS=['-g', '-O2', '-arch', 'x86_64'])
+        env.Append(CCFLAGS=['-g', '-O2', '-arch', 'x86_64', '-std=c++17'])
```

Comment:
That's wrong. As you can see for other platforms, this should be appended to `CXXFLAGS` (C++), not `CCFLAGS` (C++ **and** C).